softioc/asyncio_dispatcher.py

```diff
@@ -47,6 +48,25 @@ def close(self):
             self.__atexit = None
 
         self.__shutdown()
+    
+    def wait_for_quit(self):
+
```

Comment:
Don't really like this extra whitespace, it doesn't add any structure to the code.